### ISSUE TITLE
Unable to update

### ISSUE THREAD

**Issue by drindt** [rpm-software-management/dnf-plugin-system-upgrade#31]
This is the console output:

```
[root@localhost ~]# dnf system-upgrade download --releasever=23
Traceback (most recent call last):
  File "/usr/bin/dnf", line 36, in <module>
    main.user_main(sys.argv[1:], exit_code=True)
  File "/usr/lib/python2.7/site-packages/dnf/cli/main.py", line 185, in user_main
    errcode = main(args)
  File "/usr/lib/python2.7/site-packages/dnf/cli/main.py", line 84, in main
    return _main(base, args)
  File "/usr/lib/python2.7/site-packages/dnf/cli/main.py", line 115, in _main
    cli.configure(map(ucd, args))
  File "/usr/lib/python2.7/site-packages/dnf/cli/cli.py", line 981, in configure
    self.optparser.usage = self.optparser.get_usage()
  File "/usr/lib/python2.7/site-packages/dnf/cli/option_parser.py", line 255, in get_usage
    usage += "%-25s %s\n" % (name, summary)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 8: ordinal not in range(128)
```

**Comment by keszybz** [comment]
Have you updated all packages before doing the upgrade ('dnf update', or maybe even 'dnf distro-sync')?

**Comment by drindt** [comment]
Yes "dnf update" is done, i haven't run "dnf distro-sync".

**Comment by keszybz** [comment]
```
rpm -q dnf dnf-plugins-core python-dnf-plugins-core python-dnf-plugin-system-upgrade python2-dnf dnf-plugin-system-upgrade hawkey libsolv
```

?

**Comment by drindt** [comment]
```
dnf-0.6.4-7.fc21.noarch
Das Paket dnf-plugins-core ist nicht installiert
Das Paket python-dnf-plugins-core ist nicht installiert
Das Paket python-dnf-plugin-system-upgrade ist nicht installiert
Das Paket python2-dnf ist nicht installiert
dnf-plugin-system-upgrade-0.5.0-1.fc21.noarch
hawkey-0.5.3-3.fc21.x86_64
libsolv-0.6.14-2.fc21.x86_64
```

**Comment by drindt** [comment]
Oh just recognized that dnf itself can't run seems broken. "fedup" shows the same issues.

**Comment by yol** [comment]
This seems to be #32

**Comment by wgwoods** [comment]
yeah I'm pretty sure this is either #32 or the `LANG=de` version, [rhbz#1277895](https://bugzilla.redhat.com/show_bug.cgi?id=1277895).

Both should be fixed by commit 4bbbf1ec0c3f387249d30dd103ff4d002c17f574, so I'm going to close this for now.
